database/education.json

```diff
@@ -0,0 +1,111 @@
+{
+    "annual_salary": {
+        "attributes": {
+            "classification": "Annual Salary",
+            "description": "Gross income is the total amount you earn (typically over the course of a year) before expenses. Net income is the profit your business earns after expenses ",
+            "information": "Depending on the home price you’re aiming for, you may want to wait a year or two before you apply for a mortgage if you’ve just moved into a higher-paying role. The longer you stay in your higher-paying position, the more your lender may be willing to loan you.",
+            "note": "The amount of money you earn plays a smaller role in getting a mortgage than you might think. ",
```

Comment:
@Atos20 @BlakeDonn @timkeresey 
Even though "information" key isn't on this pull request, doesn't mean we won't be adding it.  This is new but was used for debugging our deployed database.

Reply:
Thanks for the heads up!